### PR TITLE
Updated version number and arguments of Stub URI parser

### PR DESCRIPTION
**Pull request by nipunshukla21**
Added parsing for new stub installer telemetry which records partner distribution IDs and versions. Also updated stub installer URI to version 9 to accommodate for new telemetry fields.

### CODE REVIEW

**Comment by relud** [comment]
lgtm, r? @mozilla/data-platform-infra-wg as the current code owners

File: ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParseUri.java [hunk 1]
```diff
@@ -248,7 +248,7 @@ private static class InvalidIntegerException extends InvalidUriException {
         .compile(String.format("^v(?<%s>[6-8])(-(?<%s>\\d+))?$", VERSION, FUNNELCAKE));
```
Comment (relud) [inline ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParseUri.java:248]:
```suggestion
        .compile(String.format("^v(?<%s>[6-9])(-(?<%s>\\d+))?$", VERSION, FUNNELCAKE));
```

**Review by akkomar — APPROVED** [review]

**Comment by relud** [comment]
@whd: can this be deployed to prod and @nipunshukla21 notified when it is done so that he can merge https://phabricator.services.mozilla.com/D198610

**Comment by whd** [comment]
@nipunshukla21 you should be good to merge https://phabricator.services.mozilla.com/D198610.

On the pipeline side we should have merged and deployed  https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/801 before this PR, since we would want the schemas to be up to date and not create any potentially (temporarily) unrouteable messages. I would have caught this at deploy time via a casual inspection described on https://mozilla-hub.atlassian.net/wiki/spaces/SRE/pages/27922303/Ingestion+Beam#IngestionBeam-ProdCodeDeploymentPolicy but because we're doing some Pub/Sub Lite testing we deployed the code changes before the schemas changes.

Fortunately https://phabricator.services.mozilla.com/D198610 presumably was also gating submission of the new pings so we were unlikely to run into any issues with deployment order. I re-ran probe scraper to pick up the generated schemas changes and everything has landed as of 11:13 PT.

**Pull request merged**